test/extended/marketplace/marketplace_explain.go

```diff
@@ -0,0 +1,26 @@
+package marketplace
+
+import (
+	g "github.com/onsi/ginkgo"
+	o "github.com/onsi/gomega"
+
+	exutil "github.com/openshift/origin/test/extended/util"
+)
+
+var _ = g.Describe("[Feature:Marketplace] Marketplace oc explain the opsrc & csc", func() {
+
+	defer g.GinkgoRecover()
+
+	var (
+		oc = exutil.NewCLI("marketplace", exutil.KubeConfigPath())
+	)
+
+	//OCP-23670 description info for opsrc&csc CRDs
+	g.It("ensure the opsrc & csc can be explain by oc command", func() {
+
+		explainOpsrc, _ := oc.AsAdmin().WithoutNamespace().Run("explain").Args("operatorsource").Output()
+		o.Expect(explainOpsrc).ShouldNot(o.ContainSubstring("<empty>"))
+		explainCsc, _ := oc.AsAdmin().WithoutNamespace().Run("explain").Args("catalogsourceconfig").Output()
+		o.Expect(explainCsc).ShouldNot(o.ContainSubstring("<empty>"))
+	})
```

Comment:
@emmajiafan this test looks great - but with the longterm plans of deprecating the two CRDs this would add additional work.

Thoughts @kevinrizza

Reply:
Then the related PRs can be in branch and removed from the master? I think so.

Reply:
This significantly adds complexity to the development workflow of these features, since a single pr can't be created to update these tests at the same time as the pr that modifies the source. Essentially the workflow becomes:

1. Create a PR in this repository that _turns off_ the test
2. Create a PR that modifies the feature
3. Create a PR that reenables this test with modifications

I think this workflow doesn't make much sense, and we should consider how to write these tests with the expectation that we should only be writing tests based on API guarantees -- my opinion being that a log message is _not_ something we should be guaranteeing.

Reply:
I agree with @kevinrizza - this is a shortcoming that we need to address.